playwright/e2e/crypto/toasts.spec.ts

```diff
@@ -53,3 +54,114 @@ test.describe("Key storage out of sync toast", () => {
         ).toBeVisible();
     });
 });
+
+test.describe("Turn on key storage toast", () => {
```

Comment:
```suggestion
test.describe("'Turn on key storage' toast", () => {
```

Just because I misparsed this as "Turn on (key storage toast)"